libs/utils/utils.js

```diff
@@ -840,6 +840,11 @@ export async function loadArea(area = document) {
       const { default: loadGeoRouting } = await import('../features/georoutingv2/georoutingv2.js');
       loadGeoRouting(config, createTag, getMetadata, loadBlock, loadStyle);
     }
+    const appendage = getMetadata('title-append');
+    if (appendage) {
+      const { default: titleAppend } = await import('../features/title-append/title-append.js');
+      titleAppend(appendage);
```

Comment:
In this case should we use a `.then` instead of `await` since there are no downstream dependencies?
```suggestion
      import('../features/title-append/title-append.js')
        .then((module) => module.default(appendage));
```

Reply:
That would make tests flake. I would have to check.

Reply:
Fixed at [e1ff268](https://github.com/adobecom/milo/pull/955/commits/e1ff268924ad2945a8e09644efd29c2791c4bd26)